core/trino-spi/src/main/java/io/trino/spi/metrics/Count.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.metrics;
+
+public interface Count
+{
+    double getTotal();
```

Comment:
Why `double`?

Reply:
In case a connector would like to aggregate a floating-point metric, returning `double` should work both for integer and floating-point metrics.
Should I change it to `long`?

Reply:
I think that would be a different type of metric, though. Let's change it to `long`. We can reconsider in the future if we find an actual use case for it.